docs/rules.md

```diff
@@ -100,7 +100,8 @@ Rules are sorted by severity.
 | `MANIFEST_FIELD_REQUIRED`                               | error    | A required field is missing                                                                     |
 | `MANIFEST_FIELD_INVALID`                                | error    | A field is invalid                                                                              |
 | `MANIFEST_FIELD_DEPRECATED`                             | error    | A field is deprecated                                                                           |
-| `MANIFEST_FIELD_UNSUPPORTED`                            | error    | A manifest field is not supported                                                               |
+| `MANIFEST_FIELD_UNSUPPORTED`                            | error    | A manifest field is not supported (or not supported for the extension manifest_version)         |
+| `MANIFEST_PERM_UNSUPPORTED`                             | error    | A manifest permission is not supported for the extension manifest_version                       |
```

Comment:
nit: maybe we can have `MANIFEST_PERMISSION_UNSUPPORTED`

Reply:
Sure, I think I opted for `MANIFEST_PERM_` mainly because it was shorter, but if we prefer to include PERMISSION for consistency with the other rule code related to permission, I'm fine with that.